config/templated/service_account_params.config.yaml

```diff
@@ -0,0 +1,3 @@
+- op: replace
```

Comment:
wouldn't it be better to use `add` and don't have the default value of `'[]'` for the `imagePullSecrets`?